recycler/src/main/java/com/elpassion/android/commons/recycler/items/ItemAdapter.kt

```diff
@@ -1,4 +1,4 @@
-package com.elpassion.android.commons.recycler
+package com.elpassion.android.commons.recycler.items
```

Comment:
Why? This is the purpose of having adapters. They adapt one thing to be usable by another.